src/components/CityWeatherShortcut/CityWeatherShortcut.styled.tsx

```diff
@@ -48,6 +47,14 @@ const City = styled.p`
   font-size: 20px;
   line-height: 24px;
   text-align: center;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  max-width: 80%;
+  white-space: nowrap;
```

Comment:
it should appear on `City` element

Reply:
It is applied to City component

Reply:
right, my bad github displayed it weirdly